src/compo/tempo_nc2ioda.py

```diff
@@ -187,60 +197,93 @@ def _read(self):
                 exit()
 
             # clean data
-            neg_obs = obs > 0.0
-            nan_obs = obs != np.nan
+            neg_obs = ((obs > 0.0) & (err > 0.0))
+            nan_obs = ((obs != np.nan) & (err != np.nan))
             cln = np.logical_and(neg_obs, nan_obs)
 
             # final flag before sending this to ioda engines
             flg = np.logical_and(flg, cln)
 
+            # print before compression
```

Comment:
Are these print statements here for debugging?

Reply:
I'd like to keep them here as default as the TEMPO current format is full of inconsistencies in masking. See PR description.